src/components/Dialogs/DraggableDialog/DraggableDialogComponent.tsx

```diff
@@ -24,7 +24,7 @@ export class DraggableDialogComponent extends React.Component<ResizableDialogCom
     componentDidUpdate() {
         const header = this.dd.getElementsByClassName("bp3-dialog-header");
         if (this.props.helpType && header.length > 0 && this.dd.getElementsByClassName("help-button").length === 0) {
-            const helpButton = <Button icon="help" minimal={true} onClick={() => this.props.appStore.helpStore.showHelpDrawer(this.props.helpType)}/>;
+            const helpButton = <Button icon="help" minimal={true} onClick={(ev: React.MouseEvent<HTMLButtonElement>) => this.onClickHelpButton(ev)}/>;
```

Comment:
can't you just use `onClick={this.onClickHelpButton}` after turning `onClickHelpButton` into an arrow-function? Anonymous functions should be avoided if possible in react, as they always trigger re-rendering